posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -593,10 +593,10 @@ async def execute_batch_export_insert_activity(
     inputs,
     non_retryable_error_types: list[str],
     finish_inputs: FinishBatchExportRunInputs,
-    start_to_close_timeout_seconds: int = 3600,
+    interval: str,
     heartbeat_timeout_seconds: int | None = 120,
-    maximum_attempts: int = 10,
-    initial_retry_interval_seconds: int = 10,
+    maximum_attempts: int = 15,
```

Comment:
Eventually, we should retry forever on retryable errors. What's blocking us from setting this to infinity is that some non-retryable errors are still tagged as retryable.

Reply:
In the meantime, bumping the number of retries to 15.